730 check rows vdsdata tasks unpredictable duration

gabrielwol opened the pull request:
## What this pull request accomplishes:
- Performance improvements for the check data tasks of VDS DAGs. 
- I guess there was some instability to the method `FROM large_table, LATERAL (SELECT X from small_CTE)` since it ran alternately in 10s/>24hrs. I changed this to a cross join between two very small CTE and it is running in <1s.  

## Issue(s) this solves:
- #730

## What, in particular, needs to reviewed:
- I removed the check on div_8001 (Non rescu part of the VDS network) because it is not summarized ever. Alternatively I could use the method mentioned in 2nd bullet point above but go back to querying raw table - or have one query type for checks on raw tables and one for summary tables. 

## What needs to be done by a sysadmin after this PR is merged
Nothing

tahaislam commented on dags/vds_pull_vdsdata.py:
```diff
@@ -132,19 +132,18 @@
                 sql="select/select-row_count_lookback.sql",
                 conn_id='vds_bot',
-                params={"table": 'vds.raw_vdsdata',
+                params={"table": f'vds.counts_15min_div{divid}',
```
Not specifically for this PR, but I think it's better to use either double or single quotes but not both. Most linters/standards prefer double quotes though

tahaislam commented on volumes/vds/sql/select/select-row_count_lookback.sql:
```diff
@@ -5,31 +5,33 @@
-GROUP BY lb.lookback_avg
\ No newline at end of file
+FROM today AS a,
+    lookback AS lb
```
`lb` is used in line 32. From where did this warning come? Have you tested the code? Could it be misspelled above, e.g., `1` instead of `l`?

tahaislam reviewed (commented):
Looks good to me. Just one comment on that sqlfluff warning

gabrielwol commented:
Small change: I moved the file to a common location to be used by other dags under `dags/sql` and updated the template_searchpath. I'd say this is ready to merge!
